src/pages/App.js

```diff
@@ -0,0 +1,21 @@
+import { useDispatch } from 'react-redux';
+import { Container } from 'react-bootstrap';
+import TopNav from '../components/TopNav';
+import Stockcontainer from '../container/StockContainer';
+import { fetchStocks } from '../redux/index';
+
+function App() {
+  const dispatch = useDispatch();
+  const handleOnSelect = e => {
+    dispatch(fetchStocks(e.target.value));
+  };
+
+  return (
+    <Container className="pt-5 pb-5">
+      <TopNav handleOnSelect={handleOnSelect} />
+      <Stockcontainer />
+    </Container>
+  );
+}
+
+export default App;
```

Comment:
- [ ] I got presented with a plain black screen when I tried running the application on localhost. It briefly shows the top nav bar when loading and then everything just disappears. 
Please verify that everything works well on your end and please see a screenshot of the error. I think it has to do with `StockNewsContainer`.

Error Screenshot |
:----------------:|
<img width="1170" alt="Screen Shot 2022-10-06 at 7 53 53 PM" src="https://user-images.githubusercontent.com/39523312/194406641-2d7b7ba0-eefa-45f0-852b-76db36f6e4e7.png"> |

Reply:
Sorry for inconvenience but I think that is on your side :)


![Screenshot 2022-10-07 at 1 32 22 AM](https://user-images.githubusercontent.com/30742617/194407635-d2d70c07-2c70-4c04-9ec7-5411bf284777.png)

Reply:
Thanks for your quick reply. 👍 
When you submit your project for a re-review, the next reviewer will also test the application to ensure it all works well. Let's keep our fingers crossed that their experience running your app will be different from mine.